<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>851</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="currentPath">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="NavigationPanel" name="leftNavPanel" native="true">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>700</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="NavigationPanel" name="rightNavPanel" native="true">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>700</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="copyButton">
        <property name="text">
         <string>Copy</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="moveButton">
        <property name="text">
         <string>Move</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="mkDirButton">
        <property name="text">
         <string>MkDir</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deleteButton">
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>620</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NavigationPanel</class>
   <extends>QWidget</extends>
   <header>navigation_panel.hh</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
